### ISSUE TITLE
Purpose of Procrustes Analysis

### ISSUE THREAD

**Issue by JoshuaHess12** [KrishnaswamyLab/PHATE#83]
Hi there,

I am using PHATE on data sets with much success, and I am looking to understand the purpose of the procrustes analysis between the classical MDS embedding and the metric MDS embedding in the embed_mds function. This is not necessarily an issue, but I couldn't find any documentation in the paper "Visualizing structure and transitions in high-dimensional biological data" on the matter. 

Thank you!
Josh

**Comment by scottgigante** [comment]
Hi @JoshuaHess12 ,

This was a recent addition as a result of the switch from using the [sklearn SMACOF](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.smacof.html) implementation of MDS to using a [stochastic gradient descent](https://github.com/jxz12/s_gd2) implementation. SGD is faster, but it sometimes produces results that are rotated by 90 or 180 degrees, depending on random seed. To produce a more stable output, I rotate the output to match the classic MDS initialisation using Procrustes.

**Comment by JoshuaHess12** [comment]
Hi @scottgigante ,

Great - that answered my question. Thank you!
